src/display/api.js

```diff
@@ -94,6 +88,15 @@ if (typeof PDFJSDev !== 'undefined' && PDFJSDev.test('GENERIC')) {
     });
   }) : null;
 }
+if (typeof PDFJSDev !== 'undefined' &&
+    PDFJSDev.test('PRODUCTION && !(MOZCENTRAL || FIREFOX)') &&
```

Comment:
Also, I'm kind of wondering if this condition needs to be so complicated, since it seems that `PDFJSDev.test('GENERIC')` really ought to be sufficient?

Reply:
I'm not 100% sure I must say, but I also they those are equivalent.

Reply:
Given that this is only a fallback anyway, and that (as mentioned elsewhere) `GlobalWorkerOptions.workerSrc` should always be set, I cannot see the harm in changing this.
I've also quickly tested the `CHROMIUM` build, and given that the `workerSrc` is being specified through the viewer everything works fine.